assembly/pom.xml

```diff
@@ -266,6 +266,13 @@
       <id>hive-provided</id>
       <properties>
         <hive.deps.scope>provided</hive.deps.scope>
+        <hive.jackson.scope>provided</hive.jackson.scope>
```

Comment:
Hmm, both in `hive-provided` and `hive-jackson-provided` profiles, the config value is `provided`. What's the difference between them?

Reply:
Previously, these dependencies follow `hive.deps.scope` like the following. So, we need to preserve the existing behavior.
```
- <scope>${hive.deps.scope}</scope>
+ <scope>${hive.jackson.scope}</scope>
```

Reply:
Oh, `hive-jackson-provided` has hive in compile scope and only jackson in provided scope.

Reply:
The name `hive-jackson-provided` makes me think both are provided.

Reply:
Yes!